src/sources/groups.js

```diff
@@ -152,23 +146,23 @@ class GroupSource extends Source {
   async getInfo() {
     try {
       /* attempt to fetch customisable information from S3 bucket */
-      const objects = await this._listObjects();
-      const objectKeys = objects.map((object) => object.Key);
+      const [logoResponse, overviewResponse] = await Promise.all([
+        fetch(await this.urlFor("group-logo.png"), {cache: "no-cache"}),
+        fetch(await this.urlFor("group-overview.md"), {cache: "no-cache"}),
+      ]);
 
-      let logoSrc;
-      if (objectKeys.includes("group-logo.png")) {
-        // Use pre-signed URL to allow client to fetch from private S3 bucket
-        logoSrc = S3.getSignedUrl('getObject', {Bucket: this.bucket, Key: "group-logo.png"});
-      }
+      const logoSrc = logoResponse.status === 200
+        ? await asDataUrl(logoResponse)
```

Comment:
This is back to how we used to do it... or at least the original way we were going to do it! Nice to avoid an additional fetch by the client 👍